core/src/dunit/scala/io/snappydata/cluster/SplitClusterDUnitTest.scala

```diff
@@ -128,6 +128,35 @@ class SplitClusterDUnitTest(s: String)
   }
 
   override protected def testObject = SplitClusterDUnitTest
+
+  // test to make sure that stock spark-shell works with SnappyData core jar
+  def testSparkShell(): Unit = {
+    // perform some operation thru spark-shell
+    val jars = Files.newDirectoryStream(Paths.get(s"$snappyProductDir/../distributions/"),
+      "snappydata-core*.jar")
+    val snappyDataCoreJar = jars.iterator().next().toAbsolutePath.toString
+    // SparkSqlTestCode.txt file contains the commands executed on spark-shell
+    val scriptFile: String = getClass.getResource("/SparkSqlTestCode.txt").getPath
+    val sparkShellCommand = productDir + "/bin/spark-shell  --master local[3]" +
+        " --conf spark.snappydata.connection=localhost:" + locatorClientPort +
+        " --conf spark.sql.catalogImplementation=in-memory" +
```

Comment:
Is explicit catalogImplementation setting always required for connector after Spark 2.1 merge? Need to discuss with Yogesh because this doesn't look proper.